docs/rules/comma-dangle.md

```diff
@@ -19,6 +19,7 @@ This rule takes one argument, which can be one of the following options:
 
 - `"always"` - warn whenever a missing comma is detected.
 - `"always-multiline"` - warn if there is a missing trailing comma on arrays or objects that span multiple lines, and warns if there is a trailing comma present on single line arrays or objects.
+- `"allow-multiline"` - warn whenever a trailing comma is detected on single line nodes.
```

Comment:
Yeah, you are right because `allow-multiline` doesn't say anything about what happens on single line nodes. I'm going to change it to `only-multiline`.